### DISCUSSION TITLE
Why is the first class more important than the latter?

### DISCUSSION THREAD

**Discussion by kon3ko** [tailwindlabs/tailwindcss#8120] [Help]
```
<p class="mt-1 mt-2" ></p>
```
Why does the above show margin 4 px instead of margin 8 px
Because the last class should be more important. Or am I misunderstanding ?

**Comment by RobinMalfait** [comment]
The order of your classes inside your html don't matter at all, the specificity is depending on the specificity in your css and the order of your css. We happen to generate `mt-2` after `mt-1` in yuor case.

You should not rely on this behaviour and instead only have 1 class active at a time. Did you want to make it conditional? `<p class="{{ someCondition ? 'mt-1' : 'mt-2' }}"></p>` will work better in this case.
